.travis.yml

```diff
@@ -9,4 +9,18 @@ env:
 before_script:
   - jdk_switcher use $JDK
 
-script: sbt ++$TRAVIS_SCALA_VERSION clean stack/clean 'test-only -- timefactor 10' 'stack/test-only -- timefactor 10'
+script: sbt ++$TRAVIS_SCALA_VERSION test
```

Comment:
I had a lot of problems with tests timing out and resulting in false negatives if the timefactors aren't set. I think we should use it here too.